lib/dry/types/array/member.rb

```diff
@@ -48,6 +48,13 @@ def try(input, &block)
             block ? yield(failure) : failure
           end
         end
+
+        # @api public
+        #
+        # @see Definition#to_ast
+        def to_ast
```

Comment:
This will become clearer once we get to the compiler. We may realize that `:member` node is node even needed. Don't worry about this for now.